### PR TITLE
Write up auth -> activity log page

### PR DESCRIPTION
**Pull request by hola-soy-milk**

### CODE REVIEW

File: content/4.auth/8.activity-log.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+description: Learn to audit user activity and enforce accountability using the activity log.
+---
+
+# Activity Log
+
+<!-- TODO: Image -->
+
+The activity log module provides a collective timeline of all data-changing actions taken within your project. It is accessed via the notifications tray of the sidebar. The page shares the filtering and search with the [Collection Page](/data-modeling/collections).
```
Comment (phazonoverload) [inline content/4.auth/8.activity-log.md]: > The page shares the filtering and search with the Collection Page

Really struggled to read this - took me a few times. Worth revision?

File: content/4.auth/8.activity-log.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+::callout{type="warning" url="/auth/creating-users" title="External Changes"}
+
+Read more about creating users.
+
+Directus can only track the events which actually pass through its platform. Therefore, any changes made to the database
+directly are not tracked.
+
+::
```
Comment (phazonoverload) [inline content/4.auth/8.activity-log.md]: What's going on here? I think line 13 needs removing?

File: content/4.auth/8.activity-log.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+## View an Activity Log Item
+
+<!-- TODO: ![Activity Log Default Fields](https://cdn.directus.io/docs/v9/configuration/activity-log/activity-log-20220816/activity-log-default-fields-20220816A.webp) -->
+
+Click on any item in the activity log and a side drawer will open, displaying its logged details. The following information
+is stored for each item:
+
+- **User** — The user that performed the action.
+- **Action** — The specific action taken (e.g., Create, Update, Delete, Comment, or Login).
+- **TimeStamp** — The timestamp of when the action was performed.
```
Comment (phazonoverload) [inline content/4.auth/8.activity-log.md]: Not sure TimeStamp is capitalized like this

**Review by phazonoverload — COMMENTED** [review]
Good. Question I'm not sure about - isn't this also a setting inside of flow configuration - whether to log activities or not?

**Comment by hola-soy-milk** [comment]
Thanks for the feedback @phazonoverload! Can you please take another look?

File: content/4.auth/8.accountability.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+The activity log module provides a collective timeline of all data-changing actions taken within your project. It is accessed via the notifications tray of the sidebar, and has the same filtering and search features as the [Collection Page](/data-modeling/collections).
+
+::callout{type="warning" url="/auth/creating-users" title="External Changes"}
+
+Read more about creating users.
+
+Directus can only track the events which actually pass through its platform. Therefore, any changes directly made to the database are not tracked in the activity log.
+
+::
```
Comment (phazonoverload) [inline content/4.auth/8.accountability.md]: Confused about this still 

- URL - creating users
- title - external changes
- content - bit of both

Comment (hola-soy-milk) [reply content/4.auth/8.accountability.md]: Thank you for pointing this out I don't know what happened here.

**Comment by hola-soy-milk** [comment]
Thank you for pointing out this error on my end! Should be good to go I hope

**Review by phazonoverload — APPROVED** [review]

**Pull request merged**